front/src/utils/date.ts

```diff
@@ -82,8 +82,14 @@ export const dateTimeToIso = (inputDateTime: string) => {
  */
 export const formatToIsoDate = (date: number | string, formatDate: boolean = false) => {
   const userTimeZone = dayjs.tz.guess(); // Format : 'Europe/Paris'
+
+  let parsedDate = dayjs(date);
+
+  parsedDate = parsedDate.tz(userTimeZone);
+
   const format = formatDate ? 'D/MM/YYYY HH:mm:ss' : '';
-  return dayjs.tz(date, userTimeZone).format(format);
+
+  return parsedDate.format(format);
```

Comment:
I think you can do it shorter like that (and removed parsedDate logic) : 

```suggestion
  return dayjs(date).tz(userTimeZone).format(format);
```

Reply:
Indeed, done 👍 !